ipsec/sa.c

```diff
@@ -0,0 +1,521 @@
+/*
+ * Phoenix-RTOS --- LwIP port
+ *
+ * Copyright 2016 Phoenix Systems
+ * Author: Jacek Popko, Michał Mirosław, Marek Białowąs
```

Comment:
Sorry for being pain in the... but we avoid putting polish characters into the code